### PR TITLE
Try to fix broken link

### LINKED ISSUE #483: Dead link in docs
https://substrate.dev/docs/en/tutorials/adding-a-module-to-your-runtime  mentions "other basic tutorials" which links to https://substrate.dev/docs/en/tutorials/tutorials/creating-your-first-substrate-chain.md which points to a 404 :(

### PR DESCRIPTION
**Pull request by JoshOrndorff**
This is my attempt to fix #483.

@riusricardo is this the correct way to link to the tutorials overview page?

### CODE REVIEW

**Comment by JoshOrndorff** [comment]
Sorry that's a big diff. Guess my editor cleaned up the whitespace.

File: website/versioned_docs/version-pre-v2.0-3e65111/tutorials/adding-a-module-to-your-runtime.md [hunk 1]
```diff
@@ -194,7 +194,7 @@ pub use contracts::Gas as ContractsGas;
 /* --snip-- */
 ```
 
-If you have followed our [other basic tutorials](tutorials/creating-your-first-substrate-chain.md), you may remember that we need to implement a `contracts::Trait` and also add `Contracts: contracts,` to our `construct_runtime!` macro.
+If you have followed our [other basic tutorials](/tutorials), you may remember that we need to implement a `contracts::Trait` and also add `Contracts: contracts,` to our `construct_runtime!` macro.
```
Comment (riusricardo) [inline website/versioned_docs/version-pre-v2.0-3e65111/tutorials/adding-a-module-to-your-runtime.md]: Should be enough to point as `tutorials`. No need to use the root path.

File: docs/tutorials/adding-a-module-to-your-runtime.md [hunk 2]
```diff
@@ -192,7 +192,7 @@ pub use contracts::Gas as ContractsGas;
 /* --snip-- */
 ```
 
-If you have followed our [other basic tutorials](tutorials/creating-your-first-substrate-chain.md), you may remember that we need to implement a `contracts::Trait` and also add `Contracts: contracts,` to our `construct_runtime!` macro.
+If you have followed our [other basic tutorials](/tutorials), you may remember that we need to implement a `contracts::Trait` and also add `Contracts: contracts,` to our `construct_runtime!` macro.
```
Comment (riusricardo) [inline docs/tutorials/adding-a-module-to-your-runtime.md]:
```suggestion
If you have followed our [other basic tutorials](tutorials/), you may remember that we need to implement a `contracts::Trait` and also add `Contracts: contracts,` to our `construct_runtime!` macro.
```

File: website/versioned_docs/version-pre-v2.0-3e65111/tutorials/adding-a-module-to-your-runtime.md (hunk 1 above)
Comment (riusricardo) [inline website/versioned_docs/version-pre-v2.0-3e65111/tutorials/adding-a-module-to-your-runtime.md]:
```suggestion
If you have followed our [other basic tutorials](tutorials/), you may remember that we need to implement a `contracts::Trait` and also add `Contracts: contracts,` to our `construct_runtime!` macro.
```

**Pull request merged**